integration/shareddata/scalars.go

```diff
@@ -147,17 +148,15 @@ var Doubles = &Values[string]{
 		"double-prec-min-minus":     -doubleMaxPrec - 1,
 		"double-prec-min-minus-two": -doubleMaxPrec - 2,
 
-		"double-null":               nil,
-		"double-1":                  float64(math.MinInt64 - 1),
-		"double-2":                  float64(math.MinInt64),
-		"double-3":                  float64(-123456789),
-		"double-4":                  float64(123456789),
-		"double-5":                  float64(math.MaxInt64),
-		"double-6":                  float64(math.MaxInt64 + 1),
-		"double-max-overflow":       9.223372036854776833e+18,
-		"double-max-overflow-verge": 9.223372036854776832e+18,
```

Comment:
I thought removing this would decrease test coverage of `$inc` or `$mul` query, or maybe aggregation `$sum` :thinking: